packages/python-packages/doc-warden/warden/WardenConfiguration.py

```diff
@@ -1,3 +1,9 @@
+# ------------------------------------------------------------------------
```

Comment:
FYI: The standard copyright header can be found at https://docs.opensource.microsoft.com/releasing/copyright-headers.html.

Reply:
Thanks sir. Will remember and update this.